install/kubernetes/helm/istio/charts/mixer/values.yaml

```diff
@@ -17,6 +17,15 @@ policy:
   autoscaleMax: 5
   cpu:
     targetAverageUtilization: 80
+  resources:
+    requests:
+      cpu: 1000m
```

Comment:
I am pretty sure we already have requests somewhere? maybe that was just telemetry though

Reply:
Yes, it was just telemetry . We may have missed policy before.

Reply:
@mandarjog is that value fit for policy?

Reply:
Yes, that is fine for standard use, but then ensure that the demo profile uses 250m.

Reply:
I think 250m is too high for demo, it should be the same as telemetry right?which I think is 100m. If telemetry is 250m though then I guess it's fine, just don't want to make demo too heavy

Reply:
Yes 100m !

Reply:
only concern is policy is disabled by default and will now reserve a whole CPU